Tests/ColorTests.swift

```diff
@@ -10,6 +10,27 @@ import XCTest
 import ACKategories
 
 final class ColorTests: XCTestCase {
+    
+    func testColorToHex() {
+        var hexColor: String
+        
+        let clearColor = UIColor.clear
+        hexColor = clearColor.hexString
+        XCTAssertEqual("#000000", hexColor)
+        
+        let blackColor = UIColor.black
+        hexColor = blackColor.hexString
+        XCTAssertEqual("#000000", hexColor)
+        
+        let whiteColor = UIColor.white
+        hexColor = whiteColor.hexString
+        XCTAssertEqual("#FFFFFF", hexColor)
+        
+        let greenColor = UIColor.green
+        hexColor = greenColor.hexString
+        XCTAssertEqual("#00FF00", hexColor)
+    }
```

Comment:
I'd add more tests for this as I can imagine that some cases might not work as expected (e.g. as mentioned above). Ideally a test for every type of `init` (well maybe we can skip `UIColor(cgColor:)`, `UIColor(ciColor:)` and definitely `UIColor(coder:)` but I think that other inits might be interesting. Or maybe simply turning the return type to optional would be enought, so we are sure that this extension doesn't make apps crash.

Reply:
I think that skipping some `init`s is not a good idea. We should be 100 % sure that our code is safe no matter which initializer of `UIColor` is used